rust-toolchain.toml

```diff
@@ -1,10 +0,0 @@
-[toolchain]
```

Comment:
Why was this removed ?

Reply:
I would prefer to sill keep rust-toolchain.toml, targets and profile can be left out

Reply:
@packgron I mention that in the description - basically the semi official actions in `action-rs` don't support new style of toolchain file and only works with the old style. I would prefer to limit the number of places where we have the version hardcoded and this way seemed best.

Alternatively I could have both toolchain file at the same time or (even more duplication) have only toml version and explicitly encoded the version across many invocations of actions in the build.yml

Reply:
Oh I see, than let's keep this